src/main/java/core/basesyntax/FileWriter.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class FileWriter implements core.basesyntax.interfaces.FileWriter {
+
+    public void writeNewFile(List<Fruit> fruit) throws IOException {
+        List<String> stringFruit = new ArrayList<>();
+        for (int i = 0; i < fruit.size(); i++) {
+            stringFruit.add(fruit.get(i).getTypeOfFruit());
+        }
+        Map<String, Long> counted = stringFruit.stream()
+                .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
+        java.io.FileWriter newFile = new java.io.FileWriter("file1.txt");
```

Comment:
```suggestion
        FileWriter newFile = new FileWriter("file1.txt");
```
please look through all your code and import all necessary classes

Reply:
выдает ошибку((( 
![image](https://user-images.githubusercontent.com/56148428/91893317-9fe72380-ec9c-11ea-9eac-aae93674ddc3.png)

Reply:
done